src/views/Statistic/StatisticLabel.js

```diff
@@ -36,11 +31,11 @@ StatisticLabel.propTypes = {
   /** Classes that will be added to the StatisticLabel className. */
   className: PropTypes.string,
 
-  // /** Primary content of the StatisticLabel. Mutually exclusive with the children prop. */
-  // content: customPropTypes.every([
-  //   customPropTypes.disallow(['children']),
-  //   PropTypes.node,
-  // ]),
+  /** Primary content of the StatisticLabel. Mutually exclusive with the children prop. */
+  label: customPropTypes.every([
```

Comment:
Now that we've had some time to consider this, I think `content` is the best option here.  Many components have a `content` section which is the main body of the component.  So this make sense to have the prop called "content".

Also, many components have a `text` prop (Dropdown, Container, Statistic, etc) that is already taken for another purpose.  Given this, let's use `content` everywhere.  I'll open and issue to update all other components to use `content`.